docs/api/howtos/create-project.mdx

```diff
@@ -15,7 +15,7 @@ import OperationHint from "@site/src/components/OperationHint";
 There are different ways to deploy a new project:
 
 1. When you have access to a **server** (which you will get for example when using the "Space Server" plan[^1]), you can create a new project on that server. Any projects deployed on a server will utilize the same shared resource pool. Since you pay for the server, you can deploy as many projects as resource utilization allows, without additional costs.
-2. For the future, we reserve the option to offer the creation of **stand-alone projects**. These will be projects that are _not_ part of a server, and will be deployed on a shared resource pool. You will pay for the resources used by the project, and you will be able to deploy as many projects as you want.
+2. You can use the ‘proSpace’[^2] tariff. The project is then allocated its own resources. Each project is billed individually.
```

Comment:
No code formatting for emphasis; and "tariff" means sth. like "Einfuhrzoll" in German, which is probably not the word you're looking for here.

```suggestion
2. You can use the _proSpace_[^2] plan. The project is then allocated its own resources. Each project is billed individually.
```